python/kvikio/CMakeLists.txt

```
@@ -26,29 +26,20 @@ project(
   LANGUAGES CXX CUDA
 )
 
-option(FIND_KVIKIO_CPP
-       "Search for existing KVIKIO C++ installations before defaulting to local files" OFF
-)
-
-# TODO: Should we symlink FindcuFile.cmake into python/cmake? find cuFile
-include(../cpp/cmake/Modules/FindcuFile.cmake)
-
-if(FIND_KVIKIO_CPP)
-  find_package(KvikIO "${RAPIDS_VERSION}")
-else()
-  set(KvikIO_FOUND OFF)
-endif()
-
+find_package(KvikIO "${RAPIDS_VERSION}" REQUIRED)
 find_package(CUDAToolkit REQUIRED)
 
-if(NOT KvikIO_FOUND)
-  add_subdirectory(../cpp kvikio-cpp)
-  set(cython_lib_dir kvikio)
-  install(TARGETS kvikio DESTINATION ${cython_lib_dir})
-endif()
-
+set(cython_lib_dir kvikio)
 include(rapids-cython-core)
 rapids_cython_init()
 
 add_subdirectory(cmake)
+
+# It would be better to factor nvcomp out into its own wheel. Until that is available, we vendor it
+# here.
+install_aliased_imported_targets(
+  TARGETS kvikio nvcomp::nvcomp nvcomp::nvcomp_gdeflate nvcomp::nvcomp_bitcomp DESTINATION
```

I don't think we need kvikio here.
```suggestion
  TARGETS nvcomp::nvcomp nvcomp::nvcomp_gdeflate nvcomp::nvcomp_bitcomp DESTINATION
```